dlp/deid_test.py

```diff
@@ -0,0 +1,168 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import pytest
+
+import deid
+
+harmful_string = 'My SSN is 372819127'
+parent = os.environ['GCLOUD_PROJECT']
```

Comment:
Call this 'project' instead of 'parent' to avoid confusing it with the fully qualified resource name.

Reply:
done